script/DeploySpark.s.sol

```diff
@@ -110,7 +110,6 @@ contract DeploySpark is Script {
         // 4. Deploy and configure ACL manager
 
         aclManager = new ACLManager(poolAddressesProvider);
-        aclManager.addPoolAdmin(deployer);
 
```

Comment:
Why are we removing these lines?

Reply:
It's not used anymore. We needed it when we were setting up reserves during the deploy, but now it just adds and removes it later so no point.